syntax = "proto3";

// Key value store.
service KeyValueStoreService {
	// Store value
	rpc put_value(PutValueInput) returns (Value) { }
	// Get value
	rpc get_value(GetValueInput) returns (Value) { }
}

message GetValueInput {
	string key = 1;
}

message PutValueInput {
	string key = 1;
	int64 value = 2;
}

message Value {
	int64 value = 1;
}
